.travis.yml

```diff
@@ -0,0 +1,41 @@
+language: python
+sudo: false
+env:
+  - PYTHON_VERSION=2.7
+before_install:
+  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+  - chmod +x miniconda.sh
+  - ./miniconda.sh -b
+  - export PATH=/home/travis/miniconda3/bin:$PATH
+  # Update conda itself
+  - conda update --yes conda
+install:
+  # At this moment, this plugin depends on qiita, so install everything in the same environment
+  - travis_retry conda create --yes -n qiita_env python=2.7 pip nose flake8
+    pyzmq networkx pyparsing natsort mock future libgfortran
+    'pandas>=0.18' 'scipy>0.13.0' 'numpy>=1.7' 'h5py>=2.3.1'
+  - source activate qiita_env
+  - pip install sphinx sphinx-bootstrap-theme coveralls ipython[all]==2.4.1
+  # Installing the simplify REST api branch - change once it has been merged to master
+  # - pip install https://github.com/biocore/qiita/archive/master.zip
```

Comment:
Can you uncomment this line and remove the next one? That branch was already merged.